src/Sylius/Bundle/AttributeBundle/Form/EventSubscriber/BuildAttributeValueFormSubscriber.php

```diff
@@ -58,7 +76,10 @@ public function preSetData(FormEvent $event)
             return;
         }
 
-        $this->addValueField($event->getForm(), $attributeValue->getAttribute());
+        $form = $event->getForm();
+        $attribute = $attributeValue->getAttribute();
+
+        $this->addValueField($form, $attribute);
```

Comment:
Why have you divided `$this->addValueField($event->getForm(), $attributeValue->getAttribute());` into four lines?